scripts/process/process_bar_chart.R

```diff
@@ -12,7 +12,7 @@ process.bar_chart <- function(viz){
   library(xml2)
   root <- read_xml("<g/>") 
   g.bars <- xml_add_child(root, 'g', id='year-bars')
-  g.mousers <- xml_add_sibling(g.bars, 'g', id = 'year-bars-mouser')
+  g.mousers <- xml_add_sibling(g.bars, 'g', id = 'year-bars-mouser', opacity = "0.0")
```

Comment:
I'm sure your reason is good, so then just merge it, but let me know so I know in the future.